### DISCUSSION TITLE
Redirecting stderr doesn't show the interface

### DISCUSSION THREAD

**Discussion by mattthhh** [Textualize/textual#4439] [General]
Hello, when I try to redirect the stderr of my app as this : `./main.py --debug 2> stderr.dbg` the interface doesn't show up. However, i'm pretty sure the program is running because when I hit Ctrl+C  the program is being killed or if I hit any other keybindings I binded before, I can see them in my debug file. Any ideas ?

**Comment by willmcgugan** [comment]
The interface is rendered to stderr, so redirecting isn't going to be useful.

If you need logging, see these docs:

https://textual.textualize.io/guide/devtools/#textual-log
